### ISSUE TITLE
Idea: create a script-less plain HTML report

### ISSUE THREAD

**Issue by filfreire** [serenity-bdd/serenity-core#1536]
Depending on the CI environment one uses, the reports generated in a Serenity automated check execution might not work properly, specially with the type of error like:

```
Blocked script execution in 'serenity-index.html' because the document's frame is sandboxed and the 'allow-scripts' permission is not set.
```

This is thoroughly discussed also in https://stackoverflow.com/questions/34315723/blocked-script-execution-in-url-because-the-documents-frame-is-sandboxed-and

I see this type of problem happening both for reports coming from a "cucumber+restassured" configuration as well as "cucumber+selenium".

@wakaleo do you think it's possible to create a "simple" script-less report  (for example with resource to [tacit](https://github.com/yegor256/tacit) for the visual part) ?

Basically a report that would be Plain HTML, while still supporting the "extra info" given for example in the case of a restassured cofiguration, where in the report I can still see all the REST info of a request made to an API.

@wakaleo if you don't have the bandwidth to do this now, can you give me a high-level view of what would be the cleanest way to implement this, and I can submit a PR, or create a separate project just for this.

**Comment by wakaleo** [comment]
If this is a Jenkins issue, there is a known and configurable solution to publishing rich HTML reports on Jenkins (I don't have the links handy, but it is easily googlable).

Otherwise, the serenity-emailer module produces a self-contained mailable summary of the test results. If you want to create a custom report that works differently, have a look at the implementation of the serenity-emailer module.

**Comment by filfreire** [comment]
@wakaleo thanks 👍
